lib/actions.js

```diff
@@ -115,12 +115,12 @@ actions.loadModels = function() {
  * the actual installation is invoked later as part of the "installing" phase.
  */
 actions.installDeps = function() {
-  this._skipInstall = this.options['skip-install'];
+  // yeoman-generator 0.20.x checks this.options.skipInstall
+  this.options.skipInstall = this.options['skip-install'];
 
   this.installDependencies({
     npm: true,
-    bower: false,
-    skipInstall: this._skipInstall
+    bower: false
```

Comment:
Does this need to run in browsers? If not, leaving the trailing `,` reduces noise in the commit history :-)